content/02-hello-world/02-first-app.md

```diff
@@ -33,30 +31,37 @@ Let's break it down:
 module Hello exposing (..)
 ```
 
-Is the name of the current module. This should match the file name i.e. `Hello.elm`
+This file defines an Elm module, and this line states the module’s name. The module name must match the file name; so in this case, the file must be named `Hello.elm`.
 
 ```elm
 import Html exposing (div, text)
 ```
 
-This imports functions from the Html module. `div` and `text` are functions that produce html nodes.
+This imports functions from the `Html` module. `div` and `text` are functions that produce HTML nodes.
 
 ```elm
 main =
 ```
 
-Programs in Elm start on the `main` function. The Elm runtime uses this as entry point.
+Programs in Elm start with the `main` function. The Elm runtime uses this its entry point.
 
 ```elm
-div []
-    [ text "Hello"
-    ]
+div [] [ text "Hello" ]
 ```
 
-This creates a div and text, equivalent to:
+This is the body of our `main` function. It returns a `div` containing some plain text, equivalent to the following HTML:
 
 ```html
 <div>Hello</div>
 ```
 
-The `div` function takes arguments. The first argument is a list of attributes e.g. `class`, `id`, etc. The second argument is a list of children html nodes.
\ No newline at end of file
+The `div` function takes two arguments:
+
+1. a list of attributes (e.g. `class`, `id`, etc.)
+
+    In this example, we have passed in an empty list (`[]`), so the `div` will have no attributes.
```

Comment:
Not sure what you mean. Are you saying there should be a blank line before the `2.` item in the list? My Markdown formatter doesn't require it, but I agree it makes the plain text more readable. Will add!